content/blog/2019/cve-2019-12995/index.md

```diff
@@ -28,15 +35,15 @@ Overall CVSS score: 7.5 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/E:F/RL:O/RC:C](http
 
 ## Vulnerability impact and Detection
 
-The Istio proxy is vulnerable if the following two conditions are satisfied:
+Envoy is vulnerable if the following two conditions are satisfied:
 * A JWT authentication policy is applied to it.
-* The JWT issuer (specified by `jwksUri`) uses RSA algorithm for signature verification (Note the bug is just triggered in this condition and have nothing to do with the RSA algorithm)
+* The JWT issuer (specified by `jwksUri`) uses the RSA algorithm for signature verification (Note the bug is just triggered in this condition and has nothing to do with the RSA algorithm)
```

Comment:
we strongly strive not to use the word note.  If we see the word note, we generally prefer a callout: https://istio.io/about/contribute/creating-and-editing-pages/#callouts

I'd recommend something like
```
{{< tip >}}
The RSA algorithm used for signature verification does not contain any known security vulnerability.  This CVE is triggered only when using this algorithm but is unrelated to the security of the system.
<< /tip >}}
```

Reply:
```suggestion
* The JWT issuer (specified by `jwksUri`) uses the RSA algorithm for signature verification

{{< tip >}}
The RSA algorithm used for signature verification does not contain any known security vulnerability.  This CVE is triggered only when using this algorithm but is unrelated to the security of the system.
<< /tip >}}

```

Reply:
line 44 should begin with {. (sorry had typo).